entente/landmarks/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .serialization import (  # noqa
+    deserialize_landmarks,
+    dump_landmarks,
+    load_landmarks,
+    serialize_landmarks,
+)
```

Comment:
Is this sufficient for exporting these methods?  It seemed reasonable to do this at the Landmarks level rather than at the root.